api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXTBQueries.java

```diff
@@ -104,15 +104,15 @@ public static String inTBProgramWithinReportingPeriodAtLocation(Integer tbProgra
    * @return String
    */
   public static String pulmonaryTB(
-      Integer encounterTypeId, Integer pulmonaryTBConcept, Integer yesConcept) {
+      Integer encounterTypeId, Integer otherDiagnosisConcept ,Integer pulmonaryTBConcept) {
```

Comment:
Can you please update the spec @TinoMuianga

Reply:
Spec Updated